components/report-engine/src/main/java/org/wso2telco/analytics/hub/report/engine/internel/CarbonReportEngineService.java

```diff
@@ -42,50 +45,56 @@ public CarbonReportEngineService() {
     }
 
     public void generateCSVReport(String tableName, String query, String reportName, int maxLength, String
-            reportType, String columns) {
+            reportType, String columns, String fromDate, String toDate, String sp) {
         int tenantId = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId(true);
 
-
         threadPoolExecutor.submit(new ReportEngineGenerator(tableName, query, maxLength, reportName, tenantId,
-                reportType, columns));
+                reportType, columns, fromDate, toDate, sp));
     }
-}
 
+    public void generatePDFReport(String tableName, String query, String reportName, int maxLength, String
```

Comment:
shall we remove the commented direction?

Reply:
did it already remove now?